pysat/utils/_core.py

```diff
@@ -433,3 +435,23 @@ def load_netcdf4(fnames=None, strict_meta=False, file_format=None,
         out.attrs = {}
 
     return out, mdata
+
+
+class NetworkLock(Lock):
+    def __init__(self, *args, **kwargs):
+        '''Lock manager compatible with networked file systems
+        '''
+        super(NetworkLock, self).__init__(*args, **kwargs)
+
+    def release(self):
+        '''Releases the Lock so the file system
+
+        From portalocker docs:
+          On some networked filesystems it might be needed to force
+          a `os.fsync()` before closing the file so it's
+          actually written before another client reads the file.
+        '''
```

Comment:
```suggestion
        """Releases the Lock so the file system

        From portalocker docs:
          On some networked filesystems it might be needed to force
          a `os.fsync()` before closing the file so it's
          actually written before another client reads the file.
        """
```